### PR TITLE
Reorg admin methods

### PR DESCRIPTION
**Pull request by ryangyoung**
Updates config file to match new structure.

### CODE REVIEW

File: content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Admin methods are meant only for trusted personnel in charge of keeping the serv
 Admin commands are available only if you connect to `rippled` on a host and port that the `rippled.cfg` file identifies as admin. By default, the commandline client uses an admin connection.
```
Comment (mDuo13) [inline content/references/rippled-api/admin-rippled-methods/admin-rippled-methods.md:7]: Suggestion: We can remove this file entirely and use the landing to generate it automatically. Make this first line the "blurb" of the "Admin rippled Methods" page.

File: dactyl-config.yml [hunk 2]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Server Control Methods
+        template: template-landing-children.html
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/server-control-methods/connect.md
+        html: connect.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: This needs to be an uppercase "Docs"—it's causing weirdness in the sidebar.

You probably typed it correctly, but (annoyingly) Atom seems to want to autocorrect this to lowercase. So beware of that moving forward.

File: dactyl-config.yml [hunk 3]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Server Control Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/server-control-methods/ledger_accept.md
+        html: ledger_accept.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: Should probably just find-replace a case-sensitive `funnel: docs` with `funnel: Docs` honestly

File: dactyl-config.yml [hunk 4]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory:
+        category: Admin rippled Methods
+        subcategory: Server Control Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/server-control-methods/stop.md
+        html: stop.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 5]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Server Control Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/server-control-methods/validation_seed.md
+        html: validation_seed.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 6]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        template: template-landing-children.html
+        targets:
+          - local
+
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/consensus_info.md
+        html: consensus_info.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 7]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/feature.md
+        html: feature.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 8]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/fetch_info.md
+        html: fetch_info.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 9]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/get_counts.md
+        html: get_counts.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 10]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/peers.md
+        html: peers.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 11]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/print.md
+        html: print.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 12]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/validator_list_sites.md
+        html: validator_list_sites.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 13]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/status-and-debugging-methods/validators.md
+        html: validators.html
+        funnel: docs
```
Comment (mDuo13) [inline dactyl-config.yml]: docs→Docs

File: dactyl-config.yml [hunk 14]
```diff
@@ -597,11 +597,227 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Status and Debugging Methods
+        targets:
+          - local
+
+
+
     # TODO: Subcategory landings and pages for:
```
Comment (mDuo13) [inline dactyl-config.yml]: You should remove these TODOs now

**Review by mDuo13 — CHANGES_REQUESTED** [review]
- The method files need to be updated to change their header levels. They should start at h1. Currently they start at h2. Be sure to also promote the other headers. You can do this with a regular expression find/replace in Atom:

    1. In the find-in-projecct box (Ctrl+Shift+F or similar), turn on the regular expression option (".*" icon)
    2. In the find box, enter `^## ` (lines starting with two # followed by a space)
    2. In the replace box, enter `# ` (start with # and a space)
    3. In the file/directory pattern box, enter `content/references/rippled-api/admin-rippled-methods/*`
    4. Find and replace all
    5. Change the "find" and "replace" boxes to add another # to each (so you're turning H3's to H2's now) and repeat until you don't find any headers. (Be sure to do this from low-level headers first so you don't end up promoting the same header multiple times.)

- For landings where the content is just an opening description and a list of children, I suggest doing the automatic landing page thing. Replace the `md` field with a `name` filed that has the proper name of the file and a `blurb` that has the intro paragraph. Then you can add blurbs to the child pages as desired.
    - You can pull blurbs for the methods from https://github.com/ripple/ripple-dev-portal/blob/master/content/references/rippled-api/rippled-old-monolith.md#list-of-admin-commands 
    - Maybe use the other text in the monolith file to inspire the blurbs for the landing pages

- You should delete the content from the old monolith file as you get it moved out. This helps us keep track of what content doesn't have a place in the new organization yet.

File: dactyl-config.yml (hunk 2 above)
Comment (ryangyoung) [reply dactyl-config.yml]: Ohh, I was wondering what was causing that and couldn't figure it out!

File: dactyl-config.yml (hunk 3 above)
Comment (ryangyoung) [reply dactyl-config.yml]: Done globally.

**Comment by ryangyoung** [comment]
Made all the suggested changes. One outstanding question is where to place this bit about deprecated commands:

> The following admin commands are deprecated and may be removed without further notice:
> 
> * `ledger_header` - Use the [`ledger` command](#ledger) instead.
> * `unl_add`, `unl_delete`, `unl_list`, `unl_load`, `unl_network`, `unl_reset`, `unl_score` - Use the configuration file for UNL management instead.
> * `wallet_seed` - Use [`wallet_propose`](#wallet-propose) instead.

File: dactyl-config.yml [hunk 15]
```diff
@@ -583,29 +584,261 @@ pages:
...
+        doc_type: References
+        supercategory: rippled API
+        category: Admin rippled Methods
+        subcategory: Logging and Data Management Methods
+        targets:
+          - local
+
+    -   md: references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/ledger_request.md
+        html: ledger_request.html
+        blurb: Query a peer server for a specific ledger version.
```
Comment (mDuo13) [inline dactyl-config.yml]: Nitpick: "a peer server"→"peer servers"

**Review by mDuo13 — APPROVED** [review]
Looks great now.

Regarding the deprecated commands bit, I think what makes the most sense for now is to put that into a markdown file for the "Admin rippled Methods" landing page. Something like the following:

```
# Admin rippled Methods

{{currentpage.blurb}}

The following admin commands are deprecated and may be removed without further notice:
 
* `ledger_header` - Use the [ledger method][] instead.
* `unl_add`, `unl_delete`, `unl_list`, `unl_load`, `unl_network`, `unl_reset`, `unl_score` - Use the configuration file for UNL management instead.
* `wallet_seed` - Use the [wallet_propose method][] instead.

<!--{# common link defs #}-->
{% include '_snippets/rippled-api-links.md' %}			
{% include '_snippets/tx-type-links.md' %}			
{% include '_snippets/rippled_versions.md' %}
```

File: dactyl-config.yml [hunk 16]
```diff
@@ -583,29 +584,261 @@ pages:
 
     -   name: Key Generation Methods #TODO: possible md landing content?
         html: key-generation-methods.html
+        blurb: Use these methods to generate and manage keys.
         funnel: Docs
         doc_type: References
         supercategory: rippled API
-        category: Public rippled Methods
+        category: Admin rippled Methods
```
Comment (mDuo13) [inline dactyl-config.yml:591]: Sorry about the copypaste-o's here and on line 603. Glad you caught and fixed them!

**Pull request merged**